templates/yandex-music.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+XDG_CONFIG_HOME=${XDG_CONFIG_HOME:-~/.config}
+CONFIG_FILE_NAME=yandex-music-flags.conf
```

Comment:
How about to allow to pass path to config file from command line arguments?

Reply:
Other electron applications also use the default path to the config file. For example: obsidian-bin, visual-studio-code-bin, spotify